### ISSUE TITLE
Still missing dependencies

### ISSUE THREAD

**Issue by Tereseus** [KingDad/bwa-u3-jammming-prj#2]
You need to include all the files in the project except node_modules. I am unable to load the repo because the HTML files are missing as you can see here:

![image](https://user-images.githubusercontent.com/19315896/41629653-b36784c8-73f8-11e8-8c70-e3358d0a6597.png)

**Comment by KingDad** [comment]
Removed node_modules and resubmitted, let me know if there are any further problems. Thanks so much!

**Comment by Tereseus** [comment]
Did you resubmit it from a different repo? I do not see the public folder. It should look like this but on github (excluding .gitignore and DS_Store which is automatically ignored by git):

![image](https://user-images.githubusercontent.com/19315896/41640538-3b185f64-7430-11e8-9597-6c886e0ac5ef.png)

**Comment by KingDad** [comment]
Yeah sorry the re-upload through codecademy didn't seem to be working to I
did it from my command line. It's showing up at the repository at this link
https://github.com/KingDad/bwa-u3-jammming-prj
Let me know if it's still not showing up!
Thanks so much!
-Peter

**Comment by Tereseus** [comment]
Ok great I can see the updates and sorry about this but you have to get Codecademy to resubmit it for review as I can not mark it as complete if I review it since it is not in my name anymore.

**Comment by KingDad** [comment]
No stress thank you for letting me know, I just resubmitted the repo itself
so hopefully it's working now. Thank you so much!
-Peter
